Resources/Private/Partials/Bootstrap/AjaxURL.html

```diff
@@ -0,0 +1,5 @@
+<script>
+    currentPageUid = "{f:cObject(typoscriptObjectPath: 'lib.pageUid')}";
+    typo3_forum_ajaxUrl = "{f:uri.action(action: 'main', controller: 'Ajax', arguments:{format:'json'}, additionalParams:{eID:'typo3_forum'})}";
+    typo3_forum_ajaxUrl_helpful = "{f:uri.action(action: '__typo3_forum_action__', controller: 'Post', arguments:{post:'__typo3_forum_post__'}, additionalParams:{eID:'__typo3_forum_eid__'})}";
```

Comment:
the naming could be changed, that's true, but since it is only used for the "helpful"-link, i named it that way.